### DISCUSSION TITLE
File upload default value?

### DISCUSSION THREAD

**Discussion by devhoussam1998** [filamentphp/filament#9867] [Help]
```
FileUpload::make('image')
                            ->image()
                            ->imageEditor()
                            ->default('https://dummyimage.com/900x700')
                            ->required(),
```
`->default('https://dummyimage.com/900x700')` does not appear

**Comment by devhoussam1998** [comment]
still no answer?

**Comment by maytham553** [comment]
any answers ?

**Reply by devhoussam1998** [reply]
still nothing

**Comment by renatoaldana** [comment]
Waiting for an answer

**Reply by MaapTom** [reply]
Me too

**Reply by devhoussam1998** [reply]
so do I

**Comment by suraj-webkul** [comment]
Hi everyone,  

The following code snippet won't work because it requires a relative path instead of an absolute URL:  

```php
FileUpload::make('image')
    ->image()
    ->imageEditor()
    ->default('https://dummyimage.com/900x700') // This needs to be a relative path
    ->required(),
```  

For example, you should use a path like:  
`attachments/01JKG5X0KDF5SM2YZMN9HSPNK7.png`  

I know it's a bit late, but I just wanted to share this fix. Hope it helps! 😊

**Reply by mhmdfarhan** [reply]
which filament version you use?, not working in filament3
